search-commons/src/main/java/no/unit/nva/search2/model/OpenSearchQuery.java

```diff
@@ -208,10 +208,6 @@ protected static String decodeUTF(String encoded) {
         return URLDecoder.decode(encoded, StandardCharsets.UTF_8);
     }
 
-//    protected String encodeUTF(String unencoded) {
-//        return URLEncoder.encode(unencoded, StandardCharsets.UTF_8).replace(SPACE_ENCODED, PLUS);
-//    }
-
```

Comment:
Do we need these lines?

Reply:
No, since I don't use query parameters, we don't have to decode anymore